packages/xsnap/test/test-xsnap.js

```diff
@@ -134,12 +134,15 @@ test('print - start compartment only', async t => {
     print('print:', 123);
     try {
       (new Compartment()).evaluate('print("456")');
-    } catch (_err) {
-      send('no print in Compartment');
+    } catch (err) {
+      send('err:' + err);
```

Comment:
Did you intend no space? If so
```suggestion
      send(`err:${err}`);
```
otherwise
```suggestion
      send(`err: ${err}`);
```

Reply:
eh, space vs no space didn't feel important

the whole clause is already inside a quasiliteral, so I couldn't use the normal backticks

Reply:
> the whole clause is already inside a quasiliteral, so I couldn't use the normal backticks

Oops. I didn't notice that. Nevermind.